frontend/src/properties/propertysections/FlexboxContainerPropertySection.tsx

```diff
@@ -50,7 +50,7 @@ export const FlexboxContainerPropertySection = ({
   });
 
   let children = widget.children.map((widget) => (
-    <div className={classes.children}>
+    <div className={classes.children} key={`${widget.id}-child`}>
```

Comment:
The `-child` suffix is not needed, and a unique key has already been added sinc this PR in https://github.com/eclipse-sirius/sirius-components/commit/42b96d43d54ffccb61eadf29ef801e0260bc170e#diff-a0eae5f5b1881059415310d5520663fafeb375d1d155c858301aee45749a6aa5

Reply:
I've removed the second commit which is not necessary anymore and rebased.
The first commit indeed fixed the warning when displaying checkboxes.